haiku-misc/quicktour/additional-files/quicktour-link-on-desktop.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+# Script to create a symlink of the Haiku Quick Tour
+# start page on the Desktop
+
+startPage="01-start.html"
+desktopDir="$(finddir B_DESKTOP_DIRECTORY)"
+
+systemDocDir="$(finddir B_SYSTEM_DOCUMENTATION_DIRECTORY)/quicktour/en"
+userDocDir="$(finddir B_USER_DOCUMENTATION_DIRECTORY)/quicktour/en"
```

Comment:
I think we could fetch the output of `findpaths -e B_FIND_PATH_DOCUMENTATION_DIRECTORY quicktour/en/$startPage` instead then links the first entry :)